csv2ofx/mappings/starling.py

```diff
@@ -0,0 +1,17 @@
+
+from operator import itemgetter
+from datetime import datetime
+
+
+def fixdate(ds):
+    dmy = ds.split('/')
+    # BUG (!?): don't understand but stolen from ubs-ch-fr.py
+    return '.'.join((dmy[1], dmy[0], dmy[2]))
+
+mapping = {
+    'has_header': True,
+    'date': lambda tr: fixdate(tr['Date']),
+    'amount': itemgetter('Amount (GBP)'),
```

Comment:
Starling offers accounts in at least GBP, EUR & USD and so this heading changes.
This should be fixed to cope with any of these, and also set the currency value attribute?